src/main/java/org/jvnet/hudson/plugins/port_allocator/PortAllocator.java

```diff
@@ -36,14 +40,21 @@ private PortAllocator(PortType[] ports){
         this.ports = ports;
     }
 
+    @DataBoundConstructor
+    public PortAllocator(String pool) {
```

Comment:
This might be backward incompatible. What will happen to instances already serialized by the previous version of this class?